llvm/lib/CodeGen/MachineVerifier.cpp

```diff
@@ -3211,17 +3211,22 @@ void MachineVerifier::calcRegsRequired() {
         todo.insert(Pred);
     }
 
-    // Handle the PHI node.
-    for (const MachineInstr &MI : MBB.phis()) {
-      for (unsigned i = 1, e = MI.getNumOperands(); i != e; i += 2) {
+    // Handle the PHI nodes.
+    // Note: MBB.phis() only returns range containing PHI/G_PHI instructions.
+    // MachineVerifier checks MIR post-ISel, and some backends do have their own
```

Comment:
Maybe instead "and some backends do have their own phi nodes" something like "and a backend may have a phi node instruction in the instruction set (e.g., SPIR-V backend)". The point of the suggestion is that I'm not aware of another backend with a phi node instruction.